src/components/Content/CardItem.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const CardItem = (props) => {
+  return (
+    <>
```

Comment:
You can remove this fragment, this is normally used to return a component that is rendering multiple nodes as siblings rather than a single node.